npm run new generates broken template with TypeScript errors

<!-- Hello! Please read the [Contributing Guidelines](CONTRIBUTING.md) before submitting an issue. -->

## Description of the issue

The `npm run new` generator is [documented](https://patternflyelements.org/docs/develop/create/) to produce a working template component. However, it doesn't.

### Steps to reproduce

1. `npm run new`
2. `npm start`

### Expected behavior

Component compiles and works.

### Actual behaviour

```
TODOs] Generated in 0.9676713880000002s
⠸
❌ [build:elements] exited with exit code 1. Output:

elements/pf-cool-element/pf-cool-element.ts:12:28 - error TS9017: Only const arrays can be inferred with --isolatedDeclarations.

12   static readonly styles = [styles];
                              ~~~~~~~~

  elements/pf-cool-element/pf-cool-element.ts:12:19
    12   static readonly styles = [styles];
                         ~~~~~~
    Add a type annotation to the property styles.

elements/pf-cool-element/pf-cool-element.ts:14:3 - error TS9008: Method must have an explicit return type annotation with --isolatedDeclarations.

14   render() {
     ~~~~~~

  elements/pf-cool-element/pf-cool-element.ts:14:3
    14   render() {
         ~~~~~~
    Add a return type to the method


Found 2 errors.

❌ 1 script failed.                                         
```

This shouldn't be hard to fix. I'll look at it as part of Day of Learning.

--- a/.changeset/dull-stingrays-film.md
+++ b/.changeset/dull-stingrays-film.md
@@ -1,0 +1,5 @@
+---
+"@patternfly/create-element": patch
+---
+
+Fixed type errors in create-element template

--- a/tools/create-element/package.json
+++ b/tools/create-element/package.json
@@ -11,6 +11,9 @@
   "bin": {
     "create-app": "bin/main.js"
   },
+  "contributors": [
+    "Martin Pitt <mpitt@redhat.com>"
+  ],
   "keywords": [
     "custom elements",
     "custom-elements",

--- a/tools/create-element/templates/element/element.ts
+++ b/tools/create-element/templates/element/element.ts
@@ -1,4 +1,4 @@
-import { LitElement, html } from 'lit';
+import { LitElement, html, type TemplateResult } from 'lit';
 import { customElement } from 'lit/decorators/custom-element.js';
 
 import styles from '<%= cssRelativePath %>';
@@ -9,9 +9,9 @@ import styles from '<%= cssRelativePath %>';
  */
 @customElement('<%= tagName %>')
 export class <%= className %> extends LitElement {
-  static readonly styles = [styles];
+  static readonly styles: CSSStyleSheet[] = [styles];
 
-  render() {
+  render(): TemplateResult<1> {
     return html`
       <slot></slot>
     `;

